EIPS/eip-4750.md

```diff
@@ -92,14 +93,15 @@ Under `PC_post_instruction` we mean the PC position after the entire immediate a
 
 *Note:* Code validation rules of [EIP-3670](./eip-3670.md) guarantee there is always an instruction following `CALLF` (since terminating instruction is required to be final one in the section), therefore `PC_post_instruction` always points to an instruction inside section bounds.
 
-6. Sets `current_section_index` to `code_section_index` and `PC` to `0`, and execution continues in the called section.
+7. Sets `current_section_index` to `code_section_index` and `PC` to `0`, and execution continues in the called section.
 
 #### `RETF`
 
 1. Does not have immediate arguments.
 2. If data stack has less than `caller_stack_height + types[code_section_index].outputs`, execution results in exceptional halt.
-3. Pops nothing and pushes nothing to data stack.
-4. Pops an item from return stack and sets `current_section_index` and `PC` to values from this item.
+3. Charges 8 gas.
```

Comment:
Good finding. @lightclient , QQ: do you plan to also add justification for the choice of `8` as gas cost? (e.g. comparing it to `JUMP`) or do you intend to let Author add the justification?

Reply:
I'll let the authors add justification.